content/collections/experiment-sdks/en/experiment-javascript.md

```diff
@@ -327,8 +327,29 @@ If your organization has purchased the [Accounts add-on](/docs/analytics/account
 Groups must either be included in the user sent with the fetch request (recommended), or identified with the user via a group identify call from the [Group Identify API](/docs/apis/analytics/group-identify) or via [`setGroup()` from an analytics SDK](/docs/sdks/analytics/browser/browser-sdk-2#user-groups).
 
 ```js
-await fetch({groups: {'org name': ['Amplitude']}});
```

Comment:
this is incorrect and does not work since the fetch call has to have a user id or device id pass in the user object.